myuw/dao/enrollment.py

```diff
@@ -18,18 +21,62 @@
     "SENIOR": 4,
     "GRADUATE": 5,
 }
+DEFAULT_CLASS_CODE = 6
+logger = logging.getLogger(__name__)
 
 
-DEFAULT_CLASS_CODE = 6
+def get_all_enrollments():
```

Comment:
This function was:
def get_all_enrollments():
    """
    :return: the dictionary of {Term: Enrollment}
    """
    regid = get_regid_of_current_user()
    return enrollment_search_by_regid(regid)


line 36-37 seem unnecessary.